src/plugins/discover/public/application/angular/context_app.js

```diff
@@ -43,6 +43,8 @@ getAngularModule().directive('contextApp', function ContextApp() {
       predecessorCount: '=',
       successorCount: '=',
       sort: '=',
+      time: '=',
```

Comment:
Since these new params are not set in context_app.html.  In line 56 of this file, the new state is initialized without `time` and `routing`, an therefore the request currently doesn't change when testing this PR